packages/plugin-fastlane/template/android/fastlane/Fastfile

```diff
@@ -31,6 +31,37 @@ lane :increment_build do
 end
 <% } -%>
 
+lane :increment_build_appcenter do
+  <% if(codePluginFastlane.plugin && codePluginFastlane.plugin.android && codePluginFastlane.plugin.android.appCenter) { -%>
```

Comment:
@seanirby sorry to be a bother but the whitespace before the if/else statements get included in the output as well. since we don't want any whitespace on EJS lines you can move them all the way to the left. Can you double check output on iOS and Android as I am still seeing these issues on both.

Reply:
Sorry about that, I'll fix this.